lib/samson/form_builder.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Samson
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    def input(attribute, as: :text_field, help: false, label: false)
```

Comment:
fair enough, all those options do indicate that this more than just an `<input>`